tests/admin_views/tests.py

```diff
@@ -1889,6 +1889,16 @@ def test_change_view(self):
                 self.assertContains(response, 'login-form')
                 self.client.get(reverse('admin:logout'))
 
+    def test_inline_model_form_extra_field(self):
+        add_url = reverse('admin11:admin_views_parent_add')
+        self.viewuser.user_permissions.add(
+            get_perm(Parent, get_permission_codename('add', Parent._meta)))
+        self.viewuser.user_permissions.add(
+            get_perm(Child, get_permission_codename('add', Child._meta)))
+        self.client.force_login(self.viewuser)
+        response = self.client.get(add_url)
+        self.assertContains(response, "Extra field")
+
```

Comment:
I think this test will fit better in `admin_inlines/tests.py`. You can check how we test similar things in this directory.

Reply:
@felixxm I change code and commit.